### PR TITLE
chore(endpoints): Update region tag with product prefix

### PR DESCRIPTION
**Pull request by arbrown**

### CODE REVIEW

**Comment by arbrown** [comment]
Freezing references to snippet in cl/534553109. Please do not merge until that is submitted (or let me take care of it.) Will un-freeze references once this PR has merged.

File: endpoints/getting-started/app.yaml
Comment (iennae) [inline endpoints/getting-started/app.yaml:1]: should leave the year starting at 2015 or change from 2015-2023

**Review by iennae — APPROVED** [review]
other than the copyright year change LGTM

**Pull request merged**
